e2e/scenario/formauth.spec.js

```diff
@@ -45,11 +45,19 @@ describe('FormAuth scenario test', function() {
         });
     },40000);
 
-    it('should execute auth with authentication button', () => {
```

Comment:
it's authentication, as in confirming the identity of the user.. authorization is the granting of specific rights

Reply:
the button is called "authorize", the functionality is called "re-authorization" and so the test case name must match